internal/parser/proto/imap.pb.go

```diff
@@ -2020,7 +2020,7 @@ func (x *Close) ProtoReflect() protoreflect.Message {
 	return mi.MessageOf(x)
 }
 
-// Deprecated: Use Close.ProtoReflect.Descriptor instead.
+// Deprecated: Use CloseAndSerializeOperationQueue.ProtoReflect.Descriptor instead.
```

Comment:
Did you get a bit trigger heavy with the find-replace? :D

Reply:
Yep. Fixed.